src/components/flowchart/flowchart.js

```diff
@@ -458,8 +458,8 @@ export class FlowChart extends Component {
    */
   handleChartClick = () => {
     this.props.onLoadNodeData(null);
-    // To reset URL to '/' when click outside of a node on flowchart
-    this.props.toFlowchartPage();
+    // To reset URL to current active pipeline when click outside of a node on flowchart
+    this.props.toSelectedPipeline();
```

Comment:
I think this is having an unintended consequence. When I load Viz in a fresh time and click on the flowchart background, this is what I see in my browser's history:

<img width="322" alt="image" src="https://github.com/kedro-org/kedro-viz/assets/16869061/c25fee38-c89e-44be-bee5-2724e26ceadc">

I don't think we should be pushing to history so often.

Reply:
good catch!

Reply:
hey thanks for spotting this @tynandebold, I've limited the amount of time we push to history by checking if the node has been clicked or not. This also helps with `onLoadNodeData`. It works pretty well

```
if (this.props.clickedNode) {
      this.props.onLoadNodeData(null);
      // To reset URL to current active pipeline when click outside of a node on flowchart
      this.props.toSelectedPipeline();
    }
```